python/ray/serve/tests/test_api.py

```diff
@@ -1219,6 +1219,35 @@ def __init__(self, counter):
     wait_for_condition(lambda: ray.get(counter.get_count.remote()) == 9)
 
 
+async def test_serve_label_selector_api(serve_instance_with_labeled_nodes):
```

Comment:
you won't be able to put this test in this file, because the rest of the tests in this file share a ray cluster, while your test needs a fresh ray cluster to work. can you move this test to one of the `test_standalone` files?